tests/unit/gpu_av_descriptor_post_process_positive.cpp

```diff
@@ -171,6 +171,177 @@ TEST_F(PositiveGpuAVDescriptorPostProcess, AliasImageBindingPartiallyBound) {
+    uint_image_info.imageLayout = VK_IMAGE_LAYOUT_GENERAL;
+    VkDescriptorGetInfoEXT uint_image_get_info = vku::InitStructHelper();
+    uint_image_get_info.type = VK_DESCRIPTOR_TYPE_SAMPLED_IMAGE;
+    uint_image_get_info.data.pSampledImage = &uint_image_info;
+    vk::GetDescriptorEXT(*m_device, &uint_image_get_info, descriptor_buffer_properties.sampledImageDescriptorSize,
+                         descriptor_data + image_offset + descriptor_buffer_properties.sampledImageDescriptorSize);
+
+    VkDescriptorAddressInfoEXT buffer_address_info = vku::InitStructHelper();
+    buffer_address_info.address = buffer.Address();
+    buffer_address_info.range = sizeof(OutData);
+    buffer_address_info.format = VK_FORMAT_UNDEFINED;
+
+    VkDescriptorGetInfoEXT buffer_get_info = vku::InitStructHelper();
+    buffer_get_info.type = VK_DESCRIPTOR_TYPE_STORAGE_BUFFER;
+    buffer_get_info.data.pStorageBuffer = &buffer_address_info;
+    vk::GetDescriptorEXT(*m_device, &buffer_get_info, descriptor_buffer_properties.storageBufferDescriptorSize,
+                         descriptor_data + buffer_offset);
+
+    VkMappedMemoryRange descriptor_buffer_memory_range = vku::InitStructHelper();
+    descriptor_buffer_memory_range.memory = descriptor_buffer.Memory().handle();
+    descriptor_buffer_memory_range.offset = 0u;
+    descriptor_buffer_memory_range.size = VK_WHOLE_SIZE;
+    vk::FlushMappedMemoryRanges(device(), 1, &descriptor_buffer_memory_range);
+
+    VkDescriptorBufferBindingInfoEXT buffer_binding_info = vku::InitStructHelper();
+    buffer_binding_info.address = descriptor_buffer.Address();
+    buffer_binding_info.usage = VK_BUFFER_USAGE_RESOURCE_DESCRIPTOR_BUFFER_BIT_EXT;
+
+    VkClearColorValue float_clear;
+    float_clear.float32[0] = 0.2f;
+    float_clear.float32[1] = 0.4f;
+    float_clear.float32[2] = 0.6f;
+    float_clear.float32[3] = 0.8f;
+    VkClearColorValue uint_clear;
+    uint_clear.uint32[0] = 1u;
+    uint_clear.uint32[1] = 0u;
+    uint_clear.uint32[2] = 1u;
+    uint_clear.uint32[3] = 0u;
+    VkImageSubresourceRange subresource_range = {VK_IMAGE_ASPECT_COLOR_BIT, 0u, 1u, 0u, 1u};
+
+    m_command_buffer.Begin();
+    vk::CmdClearColorImage(m_command_buffer, float_image, VK_IMAGE_LAYOUT_GENERAL, &float_clear, 1u, &subresource_range);
+    vk::CmdClearColorImage(m_command_buffer, uint_image, VK_IMAGE_LAYOUT_GENERAL, &uint_clear, 1u, &subresource_range);
+    vk::CmdBindDescriptorBuffersEXT(m_command_buffer, 1, &buffer_binding_info);
+    uint32_t buffer_index = 0u;
+    VkDeviceSize offset = 0u;
+    vk::CmdSetDescriptorBufferOffsetsEXT(m_command_buffer, VK_PIPELINE_BIND_POINT_COMPUTE, pipeline_layout.handle(), 0u, 1u,
+                                         &buffer_index, &offset);
+    vk::CmdBindPipeline(m_command_buffer, VK_PIPELINE_BIND_POINT_COMPUTE, pipe);
+    vk::CmdDispatch(m_command_buffer, 1, 1, 1);
+    m_command_buffer.End();
+    m_default_queue->Submit(m_command_buffer);
```

Comment:
```suggestion
    m_default_queue->SubmitAndWait(m_command_buffer);
```

no need for `DeviceWaitIdle`